x-pack/heartbeat/monitors/browser/source/inline.go

```diff
@@ -26,6 +28,16 @@ func (s *InlineSource) Validate() error {
 }
 
 func (s *InlineSource) Fetch() (err error) {
+	// backwards compatibility, skip decoding if the script is already decoded
+	if strings.Contains(s.Script, "step(") {
```

Comment:
It might be nice adding a note here explaining that the `(` is the reason that this works, since it's not in the list of base64 symbols

Reply:
Good point 👍🏽